src/frontend/src/pages/WorkPackageDetailPage/ActivateWorkPackageModalContainer/ActivateWorkPackageModal.tsx

```diff
@@ -91,97 +94,86 @@ const ActivateWorkPackageModal: React.FC<ActivateWorkPackageModalProps> = ({
+                className={'padding: 10'}
+                value={value}
+                shouldDisableDate={isStartDateDisabled}
+                slotProps={{ textField: { autoComplete: 'off' } }}
               />
-            </Grid>
-            <Grid item xs={6}>
-              <FormControl fullWidth>
-                <FormLabel>Start Date (YYYY-MM-DD)</FormLabel>
-                <Controller
-                  name="startDate"
-                  control={control}
-                  rules={{ required: true }}
-                  render={({ field: { onChange, value } }) => (
-                    <>
-                      <DatePicker
-                        format="yyyy-MM-dd"
-                        onChange={(date) => onChange(date ?? new Date())}
-                        className={'padding: 10'}
-                        value={value}
-                        shouldDisableDate={isStartDateDisabled}
-                        slotProps={{ textField: { autoComplete: 'off' } }}
-                      />
-                    </>
-                  )}
-                />
-              </FormControl>
-            </Grid>
-            <Grid item xs={6}>
-              <FormControl fullWidth>
-                <FormLabel>Are the WP details correct?</FormLabel>
-                <Controller
-                  name="confirmDetails"
-                  control={control}
-                  rules={{ required: true }}
-                  render={({ field: { onChange, value } }) => (
-                    <>
-                      <RadioGroup
-                        value={value}
-                        row
-                        aria-labelledby="demo-row-radio-buttons-group-label"
-                        name="row-radio-buttons-group"
-                        onChange={onChange}
-                      >
-                        <FormControlLabel value={1} control={<Radio />} label="Yes" />
-                        <FormControlLabel value={0} control={<Radio />} label="No" />
-                      </RadioGroup>
-                    </>
-                  )}
-                />
-              </FormControl>
-            </Grid>
-          </Grid>
-        </DialogContent>
-        <DialogActions>
-          <Box textAlign="right" sx={{ my: 2 }}>
-            <NERFailButton variant="outlined" form="activate-work-package-form" onClick={onHide} sx={{ mx: 1 }}>
-              Cancel
-            </NERFailButton>
-            <NERSuccessButton variant="contained" type="submit" form="activate-work-package-form" sx={{ mx: 1 }}>
-              Submit
-            </NERSuccessButton>
-          </Box>
-        </DialogActions>
-      </Dialog>
-    </form>
+            )}
+          />
+        </Grid>
+        <Grid item xs={6}>
+          <FormControl fullWidth>
+            <FormLabel>Are the WP details correct?</FormLabel>
+            <Controller
+              name="confirmDetails"
+              control={control}
+              rules={{ required: true }}
```

Comment:
(For this and the other ones) you don't need these rules because you have the schema